### PR TITLE
Update to current api

### PR DESCRIPTION
**Pull request by zifeo**
This one needs a careful review.

### CODE REVIEW

**Comment by ustroetz** [comment]
@mthh do you want to have a look at this?

File: osrm/core.py [hunk 1]
```diff
@@ -143,15 +154,14 @@ def simple_route(coord_origin, coord_dest, coord_intermediate=None,
     geometry : str, optional
         Format in which decode the geometry, either "polyline" (ie. not decoded),
         "geojson", "WKT" or "WKB" (default: "polyline").
+    annotations : str, optional
+    continue_straight : str, optional
```
Comment (mthh) [inline osrm/core.py]: It looks like your are not using the continue_straight option when building the url ?

File: osrm/core.py [hunk 2]
```diff
@@ -439,7 +456,11 @@ def trip(coords, steps=False, output="full",
          coords_request,
          '?steps={}'.format(str(steps).lower()),
          '&geometries={}'.format(geom_request),
-         '&overview={}'.format(overview)
+         '&overview={}'.format(overview),
+         '&roundtrip={}'.format(str(roundtrip).lower()),
+         '&source={}'.format(source),
+         '&destination={}'.format(destination),
+         '&annotations={}'.format(annotations)
```
[the remark this answers is not in the document]
Comment (zifeo) [reply osrm/core.py]: I left `annotations` as a string to be consistent with https://github.com/ustroetz/python-osrm/blob/master/osrm/core.py#L196. One can change everywhere if needed?

File: osrm/core.py (hunk 1 above)
Comment (zifeo) [reply osrm/core.py]: Good catch, fixed.

File: osrm/core.py [hunk 3]
```diff
@@ -187,9 +187,9 @@ def simple_route(coord_origin, coord_dest, coord_intermediate=None,
 
         url.extend([
             '{},{}'.format(coord_dest[0], coord_dest[1]),
-            "?overview={}&steps={}&alternatives={}&geometries={}".format(
+            "?overview={}&steps={}&alternatives={}&geometries={}&continue_straight={}".format(
                  overview, str(steps).lower(),
-                 str(alternatives).lower(), geom_request)
+                 str(alternatives).lower(), geom_request, continue_straight)
```
Comment (mthh) [inline osrm/core.py]: Could you also add the `annotations` option here ? 
And also add  the `continue_straight` option a few lines below (https://github.com/ustroetz/python-osrm/blob/master/osrm/core.py#L196) so the two branches of the condition will use the same options when building the url ?

File: osrm/core.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mthh) [reply osrm/core.py]: Sure, that's fine for me like that!

**Comment by zifeo** [comment]
@mthh @ustroetz Everything updated :)

**Comment by mthh** [comment]
Great! By the way it should also fix #10!

**Pull request merged**